### PR TITLE
Added Bytebot wikipedia plugin

### PR DESCRIPTION
**Pull request by petrk94**
Added wikipedia plugin

### CODE REVIEW

File: plugins/wikipedia.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+        config = BYTEBOT_PLUGIN_CONFIG['wikipedia']
+
+        if msg.find(' ') == -1:
+            irc.msg(channel, 'usage: !wiki TOPIC')
+            return
+        else:
+            request = " ".join(msg.split(' ')[1:])
+
+        url = config['url'] + request
```
Comment (mkzero) [inline plugins/wikipedia.py]: "request" needs to be URL-encoded

**Comment by mkzero** [comment]
This still cannot be merged as the Plugin contains PEP8 errors

**Comment by petrk94** [comment]
How it can still contain PEP8 error?
We have checked it last and even Travis said its good.

Am 25.10.2016 09:38 schrieb "mkzero" <notifications@github.com>:

**Comment by mape2k** [comment]
@petrk94: See https://github.com/petrk94/Bytebot/pull/1 - this should fix the last issues with pep8. Just Merge my pullrequest should add my changes to this pull request and travis will be fine.

**Comment by mape2k** [comment]
Hooray;)

@mkzero: Please check again and merge

**Pull request merged**